kubernetes/loculus/values.schema.json

```diff
@@ -0,0 +1,1005 @@
+              "description": "A URL describing the restricted data use terms."
+            }
+          },
+          "required": ["open", "restricted"],
+          "additionalProperties": false
+        }
+      },
+      "required": ["enabled", "urls"],
+      "additionalProperties": false
+    },
+    "subdomainSeparator": {
+      "type": "string",
+      "default": "-"
+    },
+    "getSubmissionListLimitSeconds": {
+      "type": "integer",
+      "default": 600
+    },
+    "replicas": {
+      "type": "object",
+      "properties": {
+        "website": {
+          "type": "integer",
+          "description": "TODO",
+          "default": 1
+        },
+        "backend": {
+          "type": "integer",
+          "description": "TODO",
+          "default": 1
+        }
+      },
+      "additionalProperties": false
+    },
+    "preprocessingTimeout": {
+      "type": "integer",
+      "description": "TODO",
+      "default": 600
+    },
+    "enaUniqueSuffix": {
+      "type": "string",
+      "description": "TODO",
+      "default": "Loculus"
+    },
+    "enaIsBroker": {
+      "type": "boolean",
+      "description": "TODO",
+      "default": false
+    },
+    "submitToEnaProduction": {
+      "type": "boolean",
+      "description": "TODO",
+      "default": false
+    },
+    "images": {
+      "type": "object",
+      "properties": {
+        "lapisSilo": {
+          "type": "string",
+          "description": "TODO",
+          "default": "ghcr.io/genspectrum/lapis-silo:0.5.2"
+        },
+        "lapis": {
+          "type": "string",
+          "description": "TODO",
+          "default": "ghcr.io/genspectrum/lapis:0.3.13"
+        }
+      },
+      "additionalProperties": false
+    },
+    "registrationTermsMessage": {
+      "type": "string",
+      "description": "TODO",
+      "default": "You must agree to the <a href=\"http://main.loculus.org/terms\">terms of use</a>."
+    },
+    "siloImportLimitSeconds": {
+      "type": "integer",
+      "description": "TODO",
+      "default": 3600
+    },
+    "gitHubEditLink": {
+      "type": "string",
+      "description": "TODO",
+      "default": "https://github.com/loculus-project/loculus/edit/main/monorepo/website/"
+    },
+    "defaultResources": {
+      "type": "object",
+      "properties": {
+        "requests": {
+          "type": "object",
+          "properties": {
+            "memory": { "type": "string" },
+            "cpu": { "type": "string" }
+          },
+          "required": ["memory", "cpu"],
+          "additionalProperties": false
+        },
+        "limits": {
+          "type": "object",
+          "properties": {
+            "memory": { "type": "string" },
+            "cpu": { "type": "string" }
+          },
+          "required": ["memory", "cpu"],
+          "additionalProperties": false
+        }
+      },
+      "required": ["requests", "limits"],
+      "additionalProperties": false
+    },
+    "resources": {
```

Comment:
sorry but this will likely need tweaking as a result of https://github.com/loculus-project/loculus/pull/3680 which was a hotfix

Reply:
Could you kindly provide a values.yaml snippet that I can use as a basis? You probably have one somewhere right?

Reply:
That PR does contain one :)  as does https://github.com/pathoplexus/pathoplexus/pull/381

Reply:
Alright! Thanks sorry I didn't see.

https://github.com/loculus-project/loculus/pull/3588/commits/030bcd25c434048345bc96f6e50dccbfee3919e2